GitUpKit/Components/GISnapshotListViewController.m

```diff
@@ -62,7 +62,7 @@ - (instancetype)initWithRepository:(GCLiveRepository*)repository {
   return self;
 }
 
-- (void)viewWillShow {
+- (void)viewWillAppear {
```

Comment:
All these NSViewController calls should be calling super, no?

Reply:
Strictly speaking, no. All of the default implementations are no-ops and none of them are marked with `NS_REQUIRES_SUPER` to emit a warning. I have no issue with adding them for safety’s sake, though.

Reply:
Addressed in next push.

Reply:
Huh, for some reason I thought it had `NS_REQUIRES_SUPER` on those calls, but I guess you're right. Anyways 😄...